drivers/timer/Kconfig.mcux_lptmr

```diff
@@ -6,11 +6,22 @@
 config MCUX_LPTMR_TIMER
 	bool "MCUX LPTMR timer"
 	default y
-	depends on DT_HAS_NXP_KINETIS_LPTMR_ENABLED
+	depends on DT_HAS_NXP_KINETIS_LPTMR_ENABLED || \
+		   MCUX_KINETIS_LPTMR
 	depends on !COUNTER_MCUX_LPTMR
 	depends on PM
 	select SYSTEM_TIMER_HAS_DISABLE_SUPPORT
 	help
 	  This module implements a kernel device driver for the NXP MCUX Low
 	  Power Timer (LPTMR) and provides the standard "system clock driver"
 	  interfaces.
+
+config MCUX_KINETIS_LPTMR
+	bool "Deprecated DT compatible"
```

Comment:
I agree. Please update this prompt or make it hidden.

Reply:
I hid it, no sense to have it as a menu config anyways